pumpkin/src/commands/mod.rs

```diff
@@ -77,6 +78,7 @@ pub fn default_dispatcher<'a>() -> CommandDispatcher<'a> {
     dispatcher.register(cmd_echest::init_command_tree());
     dispatcher.register(cmd_kill::init_command_tree());
     dispatcher.register(cmd_kick::init_command_tree());
+    dispatcher.register(cmd_say::init_command_tree());
```

Comment:
This isn't your problem per-se, but When I see large blocks of repetitive code like this my eyes start to glaze over and I stop being able to read.

It sets off alarm bells that the code could be simplified some. It might not be getting to that point right now, but with how large and repetitive this is getting, another abstraction would be nice.

### List Option
Something like 
```rust
fn register_commands(&mut self, commands: impl IntoIterator<C>) 
where C: CommandTree
{
    for command in commands {
        dispatcher.register(command::default());
    }
}

// in default_dispatcher
dispatcher.register_commands(vec![ cmd_pumpkin, cmd_gamemode, ... ]);
```
### Macro Option
```rust
macro_rules! init_commands {
    (($command:ident),*) => { $( dispatcher.register($command::init_command_tree()); )*}
}

// In default_dispatcher
init_commands!(cmd_pumpkin, cmd_gamemode, cmd_stop, cmd_help, cmd_echest, cmd_kill);
```

Say what you will about macros, but as long as they're explained, they make the boilerplate much more legible.

In the current code, `dispatcher.register(...::init_command_tree());` is all noise, and what you're actually reading needs to be that tiny `cmd_say`


The macro option can be implemented as-is, but the list list one requires you to do some more significant refactors. Neither has to be implemented right now, but it's getting there.